Gemfile

```diff
@@ -0,0 +1,24 @@
+source "https://rubygems.org"
```

Comment:
is this file required for proper rendering of the web site? If not, I suggest it not be added.